lib/widgets/amount_form_field/amount_form_field.dart

```diff
@@ -37,10 +37,12 @@ class AmountFormField extends TextFormField {
     super.enabled,
     super.onChanged,
     bool? readOnly,
+    bool? autofocus,
   }) : super(
           keyboardType: TextInputType.numberWithOptions(
             decimal: bitcoinCurrency != BitcoinCurrency.sat,
           ),
+          autofocus: autofocus ?? false,
```

Comment:
**Note:** `AmountFormField` extends `TextFormField` and it's `autofocus` value is `false` by default.